packages/opencensus-web-all/src/initial-load-context.ts

```diff
@@ -42,17 +42,16 @@ export function getInitialLoadSpanContext(): SpanContext {
     console.log(`Invalid traceparent: ${windowWithOcwGlobals.traceparent}`);
     return randomSampledSpanContext();
   }
+  // Update the sampling decision with the value coming from the traceparent.
+  const traceparentSamplingDecision = spanContext.options || 0;
+  updateSamplingDecision(traceparentSamplingDecision);
```

Comment:
Would it work to just cache the whole initial load span context rather than just the sampling decision?

My thinking is that we will need the trace ID and span ID anyway for adding the links to the initial load span, so we may as well cache the whole object.